components/Article.js

```diff
@@ -23,8 +22,7 @@ class Article extends Component {
           <MarkdownPreview value={article.text} />
         </div>
       </div>
-      <UserComments comments={article.comments} />
-      <a className='c-user-comments__back-btn'>Back to Article List</a>
+      <Link href='/index'><a className='c-user-comments__back-btn' onClick={click}>Back to Article List</a></Link>
```

Comment:
What does `onClick={click}` do here?

Reply:
I had used it in my nav, <li><Link href='/index'><a onClick={click}>Home</a></Link></li> so thought that was correct

Reply:
Not needed, removed.